integration-tests/dependencies.gradle

```diff
@@ -32,9 +33,16 @@ dependencies {
             project(path: ':fineract-savings', configuration: 'runtimeElements'),
             project(path: ':fineract-provider', configuration: 'runtimeElements'),
             project(path: ':fineract-avro-schemas', configuration: 'runtimeElements'),
-            project(path: ':fineract-client', configuration: 'runtimeElements'),
             project(path: ':fineract-progressive-loan', configuration: 'runtimeElements')
             )
+
+    // Retrofit client for existing tests
+    testImplementation project(path: ':fineract-client', configuration: 'runtimeElements')
+
+    // Feign client - exclude services to avoid conflicts with Retrofit client
```

Comment:
if you exclude the generated services, how can any of the integration tests using this client?